test/integration_test.py

```diff
@@ -401,9 +400,9 @@ def test_snapshot_listing(self):
         for page_size in 1, 2:
             with self.subTest(page_size=page_size):
                 with mock.patch.object(TDRClient, 'page_size', page_size):
-                    paged_snapshots = self._public_tdr_client.snapshot_names_by_id(filter=filter)
-                    self.assertGreaterEqual(self._public_tdr_client.page_size, 1)
-                snapshots = self._public_tdr_client.snapshot_names_by_id(filter=filter)
+                    paged_snapshots = self._tdr_client.snapshot_names_by_id(filter=filter)
+                    self.assertGreater(len(paged_snapshots), 0)
+                snapshots = self._tdr_client.snapshot_names_by_id(filter=filter)
```

Comment:
A former comment on the use of the non public TDR client for this test: https://github.com/DataBiosphere/azul/pull/3286#discussion_r683064489